Sources/SwiftCliCore/InputValidator.swift

```diff
@@ -3,7 +3,7 @@ import Foundation
 public class InputValidator {
     public init() {}
 
-    public func validate(row: String) throws -> Bool {
+    public func validateEachStitch(row: String) throws -> Bool {
```

Comment:
Note to self: Let's look at this whole class later and the tests, especially in the context of public and private methods.